src/Messages/Attachments/TSAttachmentPointer.h

```diff
@@ -17,11 +17,13 @@ typedef NS_ENUM(NSUInteger, TSAttachmentPointerState) {
  */
 @interface TSAttachmentPointer : TSAttachment
 
+- (instancetype)init NS_UNAVAILABLE;
+
 - (instancetype)initWithServerId:(UInt64)serverId
                              key:(NSData *)key
                           digest:(nullable NSData *)digest
                      contentType:(NSString *)contentType
-                           relay:(NSString *)relay NS_DESIGNATED_INITIALIZER;
```

Comment:
This was causing two warnings.  I might be wrong, but I think the init + NS_UNAVAILABLE is better?